Fix jingoo build under opam sandbox

hcarty opened the pull request:
See #15508 - prior to this change jingoo builds under opam could
(attempt to) escape their sandbox.

Fixes #15508

hcarty commented:
(hopefully fixes it anyway)

hcarty commented:
PR opened upstream too at https://github.com/tategakibunko/jingoo/pull/103

mseri commented:
Thanks! Please make this change to a new 1.3.0-1 and a new 1.3.1-1 version of the package.

mseri commented:
@tategakibunko given your PR to release version 1.3.2, are you ok with these changes?

hcarty commented:
@mseri ah, good call - I'll move these to new versions.  And I'm happy to wait for @tategakibunko's approval/rejection on this, as the new 1.3.2 release also fixes this problem for me.

hcarty commented:
Updated to include 1.3.(0,1)-1 versions and reverted the changes to 1.3.0 and 1.3.1.

This can be closed or merged, based on what @tategakibunko thinks is best!

camelus commented:
Commit: c6528832f9823722fd6efe391841f1a67031688a

A pull request by opam-seasoned @hcarty.

##### :sunny: All lint checks passed <small>c6528832f9823722fd6efe391841f1a67031688a</small>

* These packages passed lint tests: `jingoo.1.3.0-1`, `jingoo.1.3.1-1`


---


##### :sunny: Installability check (+2)

* new installable packages (2): `jingoo.1.3.0-1` `jingoo.1.3.1-1`

mseri commented:
Ping @tategakibunko

tategakibunko commented:
Sorry for late response, I'm all ok with `1.3.0-1` and `1.3.1-1`.

mseri commented:
Thanks!

hcarty commented:
Thank you @tategakibunko and @mseri!
